src/Mod/TechDraw/TechDrawTools/__init__.py

```diff
@@ -0,0 +1,31 @@
+# ***************************************************************************
+# *                                                                         *
+# *   Copyright (c) 2022 - Wanderer Fan <wandererfan@gmail.com>             *
+# *                                                                         *
+# *   This program is free software; you can redistribute it and/or modify  *
+# *   it under the terms of the GNU Lesser General Public License (LGPL)    *
+# *   as published by the Free Software Foundation; either version 2 of     *
+# *   the License, or (at your option) any later version.                   *
+# *   for detail see the LICENCE text file.                                 *
+# *                                                                         *
+# *   This program is distributed in the hope that it will be useful,       *
+# *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+# *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+# *   GNU Library General Public License for more details.                  *
+# *                                                                         *
+# *   You should have received a copy of the GNU Library General Public     *
+# *   License along with this program; if not, write to the Free Software   *
+# *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
+# *   USA                                                                   *
+# *                                                                         *
+# ***************************************************************************
+
```

Comment:
The __init__.py file needs to import all the modules of TechDrawTools

If you need to use TechDrawTools, you currently need to write: `from TechDrawTools import tool1, tool2, tool3, tool4, etc.`
By creating the __init__.py file correctly, all tools from TechDrawTools can be imported by `import TechDrawTools`

See this tutorial: https://timothybramlett.com/How_to_create_a_Python_Package_with___init__py.html

Reply:
Could not get this to work, but this "from TechDrawTools import TDToolsMovers" in TaskXXXXView.py works.